torchgeo/datasets/rioxarray.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""In-memory geographical xarray.DataArray."""
```

Comment:
Do we only support DataArray or also Dataset? Most of the time data comes bundled together with multiple layers for different features.

Reply:
The `xr.open_dataset()` function works for both I think so it should support both as far as I can tell and with the examples I have tried.